tests/foreman/api/test_convert2rhel.py

```diff
@@ -6,7 +6,7 @@
 
 :CaseLevel: Integration
 
-:CaseComponent: Convert2rhel
+:CaseComponent: Registration
```

Comment:
Registration seems irrelevant here, Can you add some description about this change?